benchmarks/test_cls.py

```diff
@@ -43,20 +43,26 @@ def set_up(request):
         sigz_1 = 0.15
         zmean_2 = 1.5
         sigz_2 = 0.15
-        z1, a1 = np.loadtxt(dirdat + "ia_amp_analytic_1.txt", unpack=True)
-        z2, a2 = np.loadtxt(dirdat + "ia_amp_analytic_2.txt", unpack=True)
+        z1, tmp_a1 = np.loadtxt(dirdat + "ia_amp_analytic_1.txt", unpack=True)
+        z2, tmp_a2 = np.loadtxt(dirdat + "ia_amp_analytic_2.txt", unpack=True)
         pz1 = np.exp(-0.5 * ((z1 - zmean_1) / sigz_1)**2)
         pz2 = np.exp(-0.5 * ((z2 - zmean_2) / sigz_2)**2)
     elif nztyp == 'histo':
         # Histogram case
         z1, pz1 = np.loadtxt(dirdat + "bin1_histo.txt", unpack=True)[:, 1:]
-        _, a1 = np.loadtxt(dirdat + "ia_amp_histo_1.txt", unpack=True)
+        _, tmp_a1 = np.loadtxt(dirdat + "ia_amp_histo_1.txt", unpack=True)
         z2, pz2 = np.loadtxt(dirdat + "bin2_histo.txt",  unpack=True)[:, 1:]
-        _, a2 = np.loadtxt(dirdat + "ia_amp_histo_2.txt", unpack=True)
+        _, tmp_a2 = np.loadtxt(dirdat + "ia_amp_histo_2.txt", unpack=True)
     else:
         raise ValueError("Wrong Nz type " + nztyp)
     bz = np.ones_like(pz1)
 
+    # Renormalize the IA amplitude to be consistent with A_IA
+    D1 = ccl.growth_factor(cosmo, 1./(1+z1))
+    D2 = ccl.growth_factor(cosmo, 1./(1+z2))
+    a1 = - tmp_a1 * D1 / (5e-14 * ccl.physical_constants.RHO_CRITICAL * cosmo['Omega_m'])
+    a2 = - tmp_a2 * D2 / (5e-14 * ccl.physical_constants.RHO_CRITICAL * cosmo['Omega_m'])
```

Comment:
A question to demonstrate my ignorance. Do we include Omega_nu here? Is that already in Omega_m?

Reply:
I think, if we had to worry about neutrinos here, we should also worry about the fact that for neutrinos the growth factor is scale dependent. @elisachisari  will know more though.

This is the reason why I don't like the cosmology dependence of the IA bias, but I agree we should use whatever is more standard.

Reply:
Right. My second question is just one of convention. Does ccl include omega nu in omega_m? This is a redshift dependent statement I suppose since they can be relativistic etc.

Reply:
Good point! I think we don't include Omega_nu in Omega_m. @c-d-leonard or @sjoudaki would know. We probably include it in the redshift-dependent density functions though.
Regarding IA, I think what we want with this renormalization is to be consistent with the literature, where I haven't seen anyone include the neutrino component so far. I can document the caveats in the note, but it's probably not worth modifying the expression for now.

Reply:
I agree with this 100%

Omega_m does not contain neutrinos ([here](https://github.com/LSSTDESC/CCL/blob/6b8f0af672098e55c151b53c9a53789de46d7176/src/ccl_core.c#L340)), but documenting this doesn't need to be done in this PR, I think.

Reply:
Briefly chatted with David and Elisa. I think we agree that in principle the massive neutrino component should be included in Omega_m in the equation above, but that its effect is small (2% in shear-intrinsic for 1 eV neutrino) and that it can be resolved later in the issue Anze opened (#647).

Reply:
resolving this for now.